docs/docs/smart-contracts/ownable.md

```diff
@@ -107,21 +96,14 @@ impl PSP1155Struct {
 
 ## Step 6: Customize your contract
 
-Customize it by adding ownable logic. We will implement `IPSP1155Mint` trait. Modifier `only_owner` will call the function for us which verifies that
-caller is the owner.
+Customize it by adding ownable logic. We will add a `owner_function` to `MyOwnable` implemenation and add the `only_owner` modifier, which will verify that the caller of the function is the owner.
 
 ```rust
-impl IPSP1155Mint for PSP1155Struct {
+impl MyOwnable {
```

Comment:
Could you put here the whole code of the contract? BEcause we don't have example for that=)
And I think will be useful to put a link on the example where are you using `Ownable`(You don't need to create a new example with Ownable, you already have it there, only put a link on it).
Do the same for `AccessContol`.